frontend/src/models/auth.ts

```diff
@@ -100,9 +101,10 @@ export default createModel<RootModel>()({
         dispatch.auth.fetchUser()
       }
     },
-    async authenticated(_: void, rootState: any) {
+    async authenticated(_: void, rootState) {
       if (rootState.auth.authenticated) {
         dispatch.auth.setBackendAuthenticated(true)
+        await cloudController.init()
```

Comment:
could you make any promise all with the await? I mean 4 await in the same method is not good

Reply:
I know some of those require they have data before the others. But I don't need the await in the case of the cloudController init, so I'll remove it.